### DISCUSSION TITLE
Role Arn

### DISCUSSION THREAD

**Discussion by remoteparas** [abuzuhri/Amazon-SP-API-CSharp#669] [General]
Since amazon has done away with IAM roles for some calls, how to make amazonconnection without Role Arn ?

**Comment by divendouk** [comment]
Also found this issue. Did anyone manage to find a workaround?

**Comment by J-W-Chan** [comment]
#667

**Comment by abuzuhri** [comment]
new version for use without Role and Use  v1.7.7
